corehq/apps/reports/templatetags/xform_tags.py

```diff
@@ -37,7 +37,7 @@ def render_form_xml(form):
     if isinstance(xml, unicode):
         xml.encode('utf-8', errors='replace')
     formatted_xml = indent_xml(xml) if xml else ''
-    return '<pre class="fancy-code prettyprint linenums"><code class="no-border language-xml">%s</code></pre>' \
```

Comment:
I think this is the only instance of this anywhere in the codebase... which means I think we can remove all the `{% include "imports/fancy-code.html" %}` everywhere, and also the dependency on `jQuery.selectText`. Hooray!